lib/device_api/android/device.rb

```diff
@@ -253,9 +253,27 @@ def diskstat
       def uptime
         ADB.get_uptime(serial)
       end
+
+      # Returns the Wifi IP address
+      def ip_address
+        network = get_network_info
+        wlan0 = network.detect { |a| a[:name] == 'wlan0' }
+        wlan0[:ip]
+      end
+
+      # Returns the Wifi mac address
+      def wifi_mac_address
+        network = get_network_info
+        wifi = network.find { |a| a[:name] == 'wlan0' }
```

Comment:
Prefer `detect` over `find`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.